MozillaFoundation/foundation.mozilla.org
# Update donate buttons to new use link for foundation.mozilla.org

# Description

This pull request updates the header and footer on foundation.mozilla.org with a new donate URL supplied.

Link to sample test page: https://foundation-s-11909-upda-tt2ngu.herokuapp.com/en/
Related PRs/issues: #11909

# To Test

Go to the homepage and inspect the donate button in the header and footer. The links should be set to /?form=donate-header and not /?form=donate
